USAGE.md

```diff
@@ -1,370 +1,707 @@
+
+Tactics depart from the SDO format with the following field:
+
+| Field | Type | Description |
+|:------|:-----|-------------|
+| `x_mitre_shortname` | string | the `x_mitre_shortname` of the tactic is used for mapping techniques into the tactic. It corresponds to `kill_chain_phases.phase_name` of the techniques in the tactic. |
 
 ### Techniques
-ATT&CK Property | ATT&CK MediaWiki | STIX Properties
---------------- | ---------------- | ---------------
-**Entry Title** | `Has technique name` | `name`
-**Tactic** | `Has tactic` | `kill_chain_phases[i].phase_name` where `kill_chain_phases[i].kill_chain_name == "mitre-attack"`
-**Description** | `Has technical description` | `description`
-**Mitigation** | `Has mitigation` | `relationship` where `relationship_type == "mitigates"`, points from a source object with `type=="course-of-action"`, which contains a `description`
-**Detection** | `Has detection` | `description` (inline heading of Detection)
-**Examples** | in software, groups as `Has technique` | `relationship`, points from the `attack-pattern` to and from `malware`, `tool`, and `intrusion-set`
-**Platform** | `Has platform` | `x_mitre_platforms`
-**Data Sources** | `Has data source` | `x_mitre_data_sources`
-**Permissions Required** | `Requires permissions` | `x_mitre_permissions_required`
-**Effective Permissions** | `Has effective permissions` | `x_mitre_effective_permissions`
-**Defense Bypassed** | `Bypasses defense` | `x_mitre_defense_bypassed`
-**System Requirements** | `Has system requirements` | `x_mitre_system_requirements`
-**Network Requirements** | `Has network requirements` | `x_mitre_network_requirements`
-**Remote Support** | `Has remote support` | `x_mitre_remote_support`
-**Contributors** | `Has contributor` | `x_mitre_contributors`
-**Impact Type** | `Not available via MediaWiki API` | `x_mitre_impact_type`
 
+A Technique in ATT&CK is defined as an [attack-pattern](http://docs.oasis-open.org/cti/stix/v2.0/csprd01/part2-stix-objects/stix-v2.0-csprd01-part2-stix-objects.html#_Toc476230921) object. 
 
-### Software
-ATT&CK Property | ATT&CK MediaWiki | STIX Properties
---------------- | ---------------- | ---------------
-**Techniques Used** | `Has technique` | `relationship` where `relationship_type == "uses"`, points to a `target` object with `type== "attack-pattern"`
-**Aliases** | `Has alias` | `x_mitre_aliases`
-**Groups** | `Has groups` | `relationship` where `relationship_type == "uses"`, points from a `source` object with `type== "intrusion-set"`
-**Contributors** | `Has contributor` | `x_mitre_contributors`
+Techniques depart from the attack-pattern format with the following fields, with domain and tactic specific fields are marked in the "applies to" column:
```

Comment:
Typo: "with domain and tactic specific fields are marked" ?

Reply:
Good catch. Updated to:
> Techniques depart from the attack-pattern format with the following fields. Domain and tactic specific fields are marked in the "applies to" column: